### ISSUE TITLE
API call is failing. Google may have changed the API response structure

### ISSUE THREAD

**Issue by sudipkumarbhattacharya** [acheong08/Bard#59]
API call is failing. Google may have changed the API response structure

**Comment by raymerjacque** [comment]
yip. i am getting the same...

**Comment by Hikoboshi00** [comment]
Have you tried updating? I don't see this issue on the new update.

**Comment by noialinkshare** [comment]
> 

Yep, I'm on the latest version already. I don't know why but it only works for the CLI version, when I run the example.py, it returns the same error. 

It was working for a while when I upgraded my version, but after leaving it for a few days (3 days or so) it just randomly returned me that error.

I discovered this when I went on the website I made with this. It kept returning Error Code: 500 Internal Server Error.

So I looked into my JS and Python code but I saw no mistakes so I decided to download the zip file provided, and it still didn't work. (By not working I mean that it's returning that error.) 

I was so frustrated that I literally resorted to ChatGPT 😂

**Comment by AndrewsPanda** [comment]
It has been down for me as well.

**Comment by 0m364** [comment]
since these aren't official apis sometimes you have to regenerate from the cookie .. it  should be working (or at least it is for me (01-july-2023 2117 EST)

**Comment by the-confused-genius** [comment]
i got the same error, so does it work with the new update?

**Comment by 0m364** [comment]
I am not getting these errors .. device galaxy s21 unrooted I will uninstall termux and try from a fresh install as I may have dependencies built into my machine that aren't listed in requirements to run . but when I make an API call it replies back with around 3 second delay
